client/src/util/ensureTargetWritable.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2020 Thomas Plougsgaard
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as path from 'path'
+import * as fs from 'fs'
+
+export default function ensureTargetWritable (workspaceFolder: string): boolean {
+  const targetPath = path.join(workspaceFolder, 'target')
+  if (!fs.existsSync(targetPath)) {
+    fs.mkdirSync(targetPath)
```

Comment:
This is not inside the try catch, so could it not throw an exception? I dont know the node api that well..

Reply:
You are correct.

https://nodejs.org/api/errors.html
> With few exceptions, Synchronous APIs (any blocking method that does not accept a callback function, such as fs.readFileSync), will use throw to report errors.

Reply:
Technically correct?

Reply:
Yes, I've updated the PR. Forgot to push before let's say.